src/middlewared/middlewared/plugins/chart_releases_linux/chart_release.py

```diff
@@ -367,13 +367,19 @@ async def do_create(self, job, data):
 
             job.set_progress(75, 'Installing Catalog Item')
 
+            file_args = []
+            if os.path.exists(os.path.join(chart_path, 'ix_values.yaml')):
+                file_args = ['-f', os.path.join(chart_path, 'ix_values.yaml')]
```

Comment:
Repeating same code in three places, wonder if there is something to do about it